packages/components/pie-modal/test/component/pie-modal.spec.ts

```diff
@@ -83,10 +93,162 @@ test.describe('`Pie Modal is closed`', () => {
                 },
             );
 
+            // Act
             await page.locator('#dialog').click();
 
+            // Assert
             expect(messages).toHaveLength(1);
         });
     });
 });
 
+test.describe('Pie Modal `isDismissible`', () => {
```

Comment:
may need to update these to `dismissible` now

Reply:
fixed.